pkgs/tools/filesystems/netatalk/default.nix

```diff
@@ -15,9 +15,14 @@ stdenv.mkDerivation rec {
   patches = [
     ./no-suid.patch
     ./omitLocalstatedirCreation.patch
+    (fetchpatch {
+      name = "make-afpstats-python3-compatible.patch"
```

Comment:
```suggestion
      name = "make-afpstats-python3-compatible.patch";
```

Reply:
thanks for catching. perks of adding a name after the build succeeds.